### PR TITLE
Add Avatar component

### PR DESCRIPTION
**Pull request by JoshuaRogan**

### CODE REVIEW

File: src/components/Avatar/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './styles.scss';
+
+const Avatar = props => (
+  <div className={`avatar ${props.className}`}>
```
Comment (vforge) [inline src/components/Avatar/index.js]: Can we go with wds-avatar instead of making our own? https://fandomdesignsystem.com/#/components/avatars

Comment (JoshuaRogan) [reply src/components/Avatar/index.js]: Yeah, interesting. I copied this from our repo. I'm going to talk with derek to make sure we are consistent on the design and then I will make the changes

**Comment by JoshuaRogan** [comment]
I updated some of the design to closer match fandom design system. There is still lots of features to add but this is a good start IMO.

**Comment by vforge** [comment]
DS Avatar is now added - please update and reopen PR if needed.

**Pull request closed without merging**